### PR TITLE
Tuning CICE for reducing ice drift and increasing a bit of ice thickn…

### PR DESCRIPTION
**Pull request by Jipingnersc**
Tuning the parameters (dragcofficient/Pstar) in CICE for TOPAZ5 reduces sea ice drift speed and increases the sea ice thickness a little.

### CODE REVIEW

**Comment by TillRasmussen** [comment]
What is the effect of changing the turning angle ?

**Review by AlfaAli — CHANGES_REQUESTED** [review]
@Jipingnersc, maybe these values should be with real of "double kind" for example
cosw = 0.9848_dbl_kind,
sinw = 0.1736_dbl_kind

**Comment by Jipingnersc** [comment]
> What is the effect of changing the turning angle ?
Hunke(2010) tested the tuning angle of 20deg which clearly adjusted the SIT pattern in the Arctic with the reduced underestimation. So in our case, it may benefit to increase the ice thickness in the Euroasian basin.

Hunke, E. C. (2010), Thickness sensitivities in the CICE sea ice model,
Ocean Modell., 34, 137–149, doi:10.1016/j.ocemod.2010.05.004

File: cice/Release-5.1/source/ice_dyn_shared.F90 [hunk 1]
```diff
@@ -38,8 +38,11 @@ module ice_dyn_shared
       real (kind=dbl_kind), parameter, public :: &
          eyc = 0.36_dbl_kind, &
                          ! coefficient for calculating the parameter E
-         cosw = c1   , & ! cos(ocean turning angle)  ! turning angle = 0
-         sinw = c0   , & ! sin(ocean turning angle)  ! turning angle = 0
+         !cosw = c1   , & ! cos(ocean turning angle)  ! turning angle = 0
+         !sinw = c0   , & ! sin(ocean turning angle)  ! turning angle = 0
+         cosw = 0.9848   , & ! cos(ocean turning angle)  ! turning angle 10 deg
+         sinw = 0.1736   , & ! sin(ocean turning angle)  ! turning angle 10 deg
```
[the remark this answers is not in the document]
Comment (Jipingnersc) [reply cice/Release-5.1/source/ice_dyn_shared.F90:44]: > 

Thanks, it can be.

**Pull request merged**